charts/cert-manager-issuers/ci/default-values.yaml

```diff
@@ -0,0 +1,25 @@
+issuers:
```

Comment:
Would issuers not require a namespace?

Reply:
And Helm will put them in it's target namespace if you don't specify one in `issuers[].metadata.namespace`. I should add an example that specify a namespace though.

Reply:
see `charts/cert-manager-issuers/examples/vault-pki-issuers-in-namespaces.yaml`